dashboard/views/_general/_report.py

```diff
@@ -2,20 +2,22 @@
 from django.contrib import messages
 from django.urls import reverse, reverse_lazy
 from django.shortcuts import render
-from django.views.generic.edit import UpdateView, DeleteView
 
 from dashboard.models import Position, Report
 from dashboard.forms import ReportForm
 from dashboard.utils import verify_brother
 
+from dashboard.views._dashboard_generic_views import DashboardUpdateView, DashboardDeleteView
+
+
 def create_report(request):
     brother = request.user.brother
-    positions = brother.position_set.exclude(title__in=['Adviser'])
+    positions = brother.position_set.exclude(title__in=['adviser'])
```

Comment:
should be using `Position.PositionChoices` here and in other instances similar to this